src/lv_font/lv_font_ap_18.c

```diff
@@ -0,0 +1,5501 @@
+#include "lvgl/lvgl.h"
```

Comment:
I explained these at the previous reply.

Reply:
> I explained these at the previous reply.

Please specify exactly what is the license of each font.
You said "free Persian font" and "Tahoma font", but where did you take them from? Could you refer us to their license?

lvgl is MIT licensed, therefore fonts distributed with it must also come with a very permissive license. You didn't specify where exactly you took the font from. If this font is copyrighted, it could be a legal problem.

Reply:
OK, sorry, I couldn't find out these fonts license type. This font must be removed.